custom_components/solarfocus/sensor.py

```diff
@@ -564,6 +568,27 @@ def native_value(self):
         device_class=SensorDeviceClass.ENUM,
         options=list(range(0, 2)),
     ),
+    SensorEntityDescription(
+        key="pellet_usage_last_fill",
+        native_unit_of_measurement=UnitOfMass.KILOGRAMS,
+        icon="mdi:gradient-vertical",
+        device_class=SensorDeviceClass.WEIGHT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="pellet_usage_total",
+        native_unit_of_measurement=UnitOfMass.KILOGRAMS,
+        icon="mdi:alpha-t-box",
+        device_class=SensorDeviceClass.WEIGHT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="heat_energy_total",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        icon="mdi:fire-circle",
```

Comment:
```suggestion
        icon="mdi:meter-gas",
```
for consistency with the other thermal energy meters